pkg/api/clustermanager.go

```diff
@@ -0,0 +1,155 @@
+package api
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// SyncSetList represents a list of SyncSets for a given cluster.
+type SyncSetList struct {
+	Syncsets []*SyncSet `json:"value"`
+	// The link used to get the next page of operations.
+	NextLink string `json:"nextLink,omitempty"`
+}
+
+type ClusterManagerConfigurationList struct {
+	ClusterManagerConfigurations []*ClusterManagerConfiguration `json:"value"`
+
+	NextLink string `json:"nextLink,omitempty"`
+}
+
+// ClusterManagerConfiguration represents the configuration from OpenShift Cluster Manager (OCM)
+type ClusterManagerConfiguration struct {
+	MissingFields
+
+	// ID is the unique identifier for the cluster manager configuration
+	ID                string                                `json:"id,omitempty"`
+	Name              string                                `json:"name,omitempty"`
+	ClusterResourceId string                                `json:"clusterResourceId,omitempty"`
+	Deleting          bool                                  `json:"deleting,omitempty"` // https://docs.microsoft.com/en-us/azure/cosmos-db/change-feed-design-patterns#deletes
```

Comment:
this field is within the clustermanagerconfiguration document as well as within this struct.  Which one is used?